en/components/grids_templates/sorting.md

```diff
@@ -124,6 +124,127 @@ You can provide @@igComponent's remote sorting by subscribing to [`onDataPreLoad
 <div class="divider--half"></div>
 }
 
+### Styling
+
+To get started with sorting styling, we need to import the `index` file, where all the theme functions and component mixins live:
+
+```scss
+// custom-grid-paging-style.component.scss
+@import '~igniteui-angular/lib/core/styles/themes/index';
+``` 
+
+Following the simplest approach, we create a new theme that extends the [`igx-grid-theme`]({environment:sassApiUrl}/index.html#function-igx-grid-theme) and accepts the `$sorted-header-icon-color` parameter.
+
+```scss
+$custom-theme: igx-grid-theme(
+    $sorted-header-icon-color: blue
+);
+```
+The last step is to **include** the component mixins: 
+
+```scss
+ @include igx-grid($custom-theme);
+```
+
+>[!NOTE]
+ >If the component is using an [`Emulated`](../themes/component-themes.md#view-encapsulation) ViewEncapsulation, it is necessary to `penetrate` this encapsulation using `::ng-deep`:
+
+ ```scss
+:host {
+  ::ng-deep {
+    @include igx-grid($custom-theme);
+  }
+}
+```
+
+#### Defining a color palette
+
+Instead of hardcoding the color values like we just did, we can achieve greater flexibility in terms of colors by using the [`igx-palette`]({environment:sassApiUrl}/index.html#function-igx-palette) and [`igx-color`]({environment:sassApiUrl}/index.html#function-igx-color) functions.
+
+`igx-palette` generates a color palette based on the primary and secondary colors that are passed:
+
+```scss
+$primary-color: #09f;
+$secondary-color: #e41c77;
+
+$light-palette: igx-palette($primary: $primary-color, $secondary: $secondary-color);
+```
+
+And then with [`igx-color`]({environment:sassApiUrl}/index.html#function-igx-color) we can easily retrieve color from the pallete. 
```

Comment:
**palette**, not `pallete`.  Also add a code snippet to demonstrate how the colors are retrieved from the palette